src/core/annotation.js

```diff
@@ -2072,6 +2072,36 @@ class ButtonWidgetAnnotation extends WidgetAnnotation {
     return newRefs;
   }
 
+  _getDefaultCheckedAppearance(params) {
+    const width = this.data.rect[2] - this.data.rect[0];
+    const height = this.data.rect[3] - this.data.rect[1];
+    const fontSize = Math.min(width, height) * 0.9;
```

Comment:
Where does this magical value come from, and is there any way to handle this without a hard-coded value?
If this is indeed necessary, can we perhaps at least replace it with a constant instead (and add a comment about how the value was chosen)?